specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -312,12 +306,12 @@ model BlocklistPolicy {
   haltOnBlocklistHit?: boolean;
 }
 
-@added(ContentSafety.Versions.v2023_10_15_Preview)
-@doc("Judgement kind.")
-enum JudgmentKind {
-  @doc("Treat all content as accepted no matter what the predicted value is.")
-  AcceptAll,
-  @doc("Accept only when the predicted severity is less than the severity threshold.")
-  AcceptLessThan,
-}
+// @added(ContentSafety.Versions.v2023_10_15_Preview)
+// @doc("Judgement kind.")
+// enum JudgmentKind {
+//   @doc("Treat all content as accepted no matter what the predicted value is.")
+//   AcceptAll,
+//   @doc("Accept only when the predicted severity is less than the severity threshold.")
+//   AcceptLessThan,
+// }
```

Comment:
It's in source control, so please just remove this instead.

Reply:
Code has been removed